content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,181 @@
+
+## Usage
+In order for this language to be **found everywhere** it's important that it is defined and understood **by everyone**.  The best and easiest way to accomplish this is to ensure that all disciplines are involved in as much user-facing communication as is reasonable to stay productive.  It's not necessary that the entire development team for your application sit in every single user-interview, however on the other hand it is not of value for the development team to be involved in any.  Translation and context sharing has a cost, and one that should not be ignored.  Engineers, especially more senior ones, should know the business domain as well as the PM and designers.
+
+### Product
+Product Managers(PMs) and designers generally interface with the end-users and stakeholders on a more regular basis that engineering.  As advocates of those individuals, PMs generally tend to "speak the same language" as them.  Their ability to remain fluent in the business language can be relatively simple, however the business language isn't necessarily the same as the ubiquitious language for the team.  Since the ubiquitous language is an agreement with the entire team including the business, there are a few things they must do to remain good stewards of the ubiquitous language for the team.
+
+The first is prematurely translating terms or meanings so their team "understand" it better.  At the end of the day, it will be better to take the time to explain the business domain, terms, and meanings to every member of the team than to try to spare them or yourself.  Words mean things. They also often mean different things to different people with different perspectives.  Using the same terminology as the business, at least at first, will ensure that all perspectives have the same understanding of the underlying meaning of it prior to agreeing upon the correct term or meaning for that thing.
+
+The second thing PMs should avoid doing is being authoritative about the ubiquitous language.  In order for the language to be ubiquitous, _everyone_ must agree on the meaning.  The business, product, design, and engineering.  This means that there is room for negotiation on the terms being used.  Every term in the ubiquitous language will have clear meaning to the business, but that does not mean that it will have a 1:1 correlation (every time) with the terms used in the business.  For example the user of the online shopping business may use the word Shopping Cart consistently, however the team may determine for one reason or another to use the term "Cart" instead.  As long as the business clearly understands that "cart" is the same as "shopping cart" with no ambiguity, then it can used in place of "shopping cart" in the ubiquitious language.
```

Comment:
You may want to pull on your note above about ubiquitous language evolving over time. That's a good reason to keep them as placeholders for a conversation with the whole team and encourage sharing new information that might lead to the team agreeing an evolution has occurred.